README.md

```diff
@@ -1,2 +1,67 @@
-# ros_qualisys
-ROS wrapper around the https://github.com/qualisys/qualisys_cpp_sdk package
+ros_qualisys
+============
+
+This is a ROS wrapper around the https://github.com/qualisys/qualisys_cpp_sdk package.
+
+## Getting started
+
+### Dependencies
+
+All dependencies are listed in the [package.xml](https://github.com/Gepetto/ros-qualisys/blob/main/package.xml) file.
+
+For building the package:
+- git
+- doxygen
+- cmake
+
+For executing the package:
+- catkin + roscpp (if with ROS1)
+- ament_cmake + rclcpp (if with ROS2)
```

Comment:
Looking at the CMake this affirmation does not seem right...
I don't think ROS2 is handled here is it?